src/images/icons/fullstack/Jamstack_Icon_Original.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="132" height="147" viewBox="0 0 132 147" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Keep the filename `jamstack.svg` for this file.

Reply:
yeah got it, fill=none. Thank you

Reply:
You misunderstood. Change the filename from `Jamstack_Icon_Original.svg` to `jamstack.svg`.